crates/chainspec/src/spec.rs

```diff
@@ -30,6 +30,8 @@ use reth_network_peers::{
     base_nodes, base_testnet_nodes, holesky_nodes, mainnet_nodes, op_nodes, op_testnet_nodes,
     sepolia_nodes,
 };
+#[cfg(feature = "telos")]
```

Comment:
Initially we will not have the reth nodes doing any peering, the Telos node itself peers on the network and then reth is just consuming the blocks from that node.  Possibly after do our initial release we will want to find a way to run reth without a Telos node, which could use peering, but we haven't planned for that yet.

So, if this is needed for reth to function then that's good, but if this is only needed to allow reth to peer with other reth nodes, we can skip this change for the initial release.

Reply:
This would apply to all the other changes related to peers and bootnodes.